### PR TITLE
Add ability to POST language executables via the API.

### PR DESCRIPTION
**Pull request by meisterT**
This makes preparation of contests easier without modifying DOMjudge's source code to update default settings.

### CODE REVIEW

File: webapp/src/Controller/API/LanguageController.php [hunk 1]
```diff
@@ -59,6 +64,64 @@ public function singleAction(Request $request, string $id): Response
         return parent::performSingleAction($request, $id);
     }
 
+    /**
+     * Get the given language for this contest.
```
Comment (nickygerritsen) [inline webapp/src/Controller/API/LanguageController.php:68]: Get seems not right

**Review by nickygerritsen — APPROVED** [review]
We might want to add a test?

**Comment by nickygerritsen** [comment]
I like the idea. This also makes #2108 obsolete

**Comment by meisterT** [comment]
> We might want to add a test?

Done

**Pull request merged**